features/content_negotiation.feature

```diff
@@ -75,34 +76,22 @@ Feature: Content Negotiation support
     {"name": "Sent in JSON"}
     """
     Then the response status code should be 201
-    And the header "Content-Type" should be equal to "application/xml"
+    And the header "Content-Type" should be equal to "application/xml; charset=utf-8"
     And the response should be equal to
     """
     <?xml version="1.0"?>
     <response><id>/dummies/2</id><description/><dummy/><dummyBoolean/><dummyDate/><dummyPrice/><relatedDummy/><relatedDummies/><jsonData/><name_converted/><name>Sent in JSON</name><alias/></response>
     """
 
+  Scenario: Requesting a the same format in the Accept header and in the URL should work
```

Comment:
What about cases where there's a mismatch? Which one takes precedence, or does it fail?